google/cloud/pubsub/internal/tracing_batch_sink_test.cc

```diff
@@ -308,10 +308,11 @@ TEST(TracingBatchSink, AsyncPublishLargeBatch) {
   auto mock = std::make_shared<pubsub_testing::MockBatchSink>();
   EXPECT_CALL(*mock, AddMessage(_)).Times(batch_size);
   EXPECT_CALL(*mock, AsyncPublish)
-      .WillOnce([](google::pubsub::v1::PublishRequest const& request) {
-        EXPECT_THAT(request, IsProtoEqual(MakeRequest(batch_size)));
-        return make_ready_future(make_status_or(MakeResponse(request)));
-      });
+      .WillOnce(
+          [&batch_size](google::pubsub::v1::PublishRequest const& request) {
```

Comment:
`clang-tidy` seems unhappy. You could use `auto constexpr kBatchSize = kDefaultMaxLinks + 1;` above

Reply:
I would also suggest capturing scalars by value (when not mutable), just like you would for a function parameters.

Reply:
Done, using the constexpr value